storage/archivist/src/main/scala/uk/ac/wellcome/platform/archive/archivist/flow/NotificationMessageFlow.scala

```diff
@@ -37,7 +37,7 @@ object NotificationMessageFlow {
           val progressUpdate = ProgressEventUpdate(
             id = bagRequest.id,
             events = List(
-              ProgressEvent(s"Started work on ingest: ${bagRequest.id}")
+              ProgressEvent(s"Started work on ingest")
```

Comment:
doesn't need to be an s-string now

```suggestion
              ProgressEvent("Started work on ingest")
```

Reply:
Oh yes, thanks